pkg/apis/serving/v1beta1/route_types.go

```diff
@@ -118,7 +119,11 @@ const (
 // are not generally shared.  This is defined separately and inlined so that
 // other types can readily consume these fields via duck typing.
 type RouteStatusFields struct {
-	// TODO(mattmoor): Domain (if not in addressable)
+	// URL holds the url that will distribute traffic over the provided traffic targets.
+	// It generally has the form {route-name}.{route-namespace}.{cluster-level-suffix}
+	// +optional
+	URL apis.URL `json:"url,omitempty"`
```

Comment:
Why a pointer in traffic target and v1alpha1, but not here?

Reply:
I think I fixed this in the change built on this 🤔